extra/changelog-generator/changelog-generator

```diff
@@ -47,8 +45,18 @@ def aggregate_dependabot_changelogs(entry_list: List[str]) -> List[str]:
                             )
                         ]
                     )
-                except IndexError:  # it may happen that only first line is available
-                    aggregated_dependabot_changelogs += f"\n* {entry}"
+                except:
+                    possible_problems.append(
+                        (
```

Comment:
Because this string style is used every other place where we append to `POSSIBLE_PROBLEMS` and also cuz multiline strings on default will include indentation unless you call some `dedent` function on that string.